src/api/schema/components/transform.rs

```diff
@@ -63,6 +67,27 @@ impl Transform {
         &*self.get_component_type()
     }
 
+    /// Transform output streams
+    pub async fn outputs(&self) -> Vec<Output> {
```

Comment:
With the definition of `outputs` being the same for sources and transforms, I think this could be DRY'd up with a helper func.